README.md

```diff
@@ -1,2 +1,55 @@
-# the-experts
-An app for connecting contractors with users
+##### the-experts
+
+## [ OUR HOSTED Experts {Not Working For NOW} ](https://the-experts.herokuapp.com/)
+
+## Local installation
+```
+ git clone https://github.com/FACN7/the-experts.git
+ cd the-experts
+ npm i 
+```
+In the project directory AND in the the-experts-fe directory, you have to run:
+
+### `npm start`
+
+Runs the app in the development mode.<br />
```

Comment:
Won't this start the Express server on Port 9000? What about starting the React server? You could make one command in top level `package.json` to start them both at same time using the [`concurrently`](https://www.npmjs.com/package/concurrently) npm package.